wireapp/wire-android-reloaded
# feat: upgrade federated assets source (AR-1563)

<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/AR-1563" title="AR-1563" target="_blank"><img alt="Story" src="https://wearezeta.atlassian.net/rest/api/2/universal_avatar/view/type/issuetype/avatar/10815?size=medium" />AR-1563</a>  I want to be able to send and receive assets to/from other backends
  </td></table>
  <br />
 
























----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?

### Issues

Modifications required since some core lib module changed, namely: AssetsId Plain to Qualified.
Enabling interop in federated environments

### Dependencies (Optional)

https://github.com/wireapp/kalium/pull/578/files

Needs releases with: https://github.com/wireapp/kalium/pull/578/files


### Testing

#### How to Test

You should, Login in a federated environment and be able to send and receive assets from external endpoints

----
#### PR Post Submission Checklist for internal contributors (Optional)

----
#### PR Post Merge Checklist for internal contributors

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.
